test/e2e/common_suite_test.go

```diff
@@ -249,16 +275,93 @@ func (tc *testCase) run() {
 
 func newTestCase(t *testing.T, testName string, assert CloudAssert, assessMessage string) *testCase {
 	testCase := &testCase{
-		testing:       t,
-		testName:      testName,
-		assert:        assert,
-		assessMessage: assessMessage,
-		podState:      v1.PodRunning,
+		testing:        t,
+		testName:       testName,
+		assert:         assert,
+		assessMessage:  assessMessage,
+		podState:       v1.PodRunning,
+		imagePullTimer: false,
 	}
 
 	return testCase
 }
 
+func reverseSlice(slice []string) []string {
```

Comment:
Can we use the `slices.Reverse` function rather than write a custom function to reverse s string slice https://pkg.go.dev/slices@master#Reverse?

Reply:
```
test/e2e/common_suite_test.go:13:2: package slices is not in GOROOT (/usr/local/go/src/slices)
FAIL    github.com/confidential-containers/cloud-api-adaptor/test/e2e [setup failed]
FAIL
make: *** [test-e2e] Error 1
```

Reply:
I think slices.Reverse might be go 1.21+ which we aren't on yet?